maintainers/maintainer-list.nix

```diff
@@ -2502,6 +2502,12 @@
     githubId = 7047019;
     name = "Florent Becker";
   };
+  gamb = {
+    email = "adam.gamble@pm.me";
+    github = "gamb";
+    githubId = 293586;
+    name = "Adam Gamble";
+  };
```

Comment:
@gamb please check if this info is correct.

Reply:
@turboMaCk Thanks, looks goods 👍